app/views/categories/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<header class="categories-header">
+    <a href="<%=categories_path %>" ><img src="/images/arrow_back.png" alt="" class="arrow-back"/></a>
+    <h2 class="big-title">ADD CATEGORY</h2>
+</header>
+
+<%= form_with model: @group, url: categories_path, method: :post, data: { turbo: false } do |form| %>
+  <div class="field top-field">
+    <%= form.label :icon %>
+    <%= form.text_field :icon, class: 'form-control', placeholder: 'http://Icon.png' %>
+  </div>
+
+  <div class="field">
+    <%= form.label :name %>
+    <%= form.text_field :name, class: 'form-control', placeholder: 'Category name' %>
+  </div>
+
+   <div class="actions">
+    <%= form.submit "Save" , class: 'login-btn'%>
+  </div>
```

Comment:
- [ ] Kindly style the save button on the forms, currently this is not visible on the page please make the button for saving responses visible by adding appropriate styling 👍🏿